src/xdotoolify.js

```diff
@@ -326,6 +326,45 @@ _Xdotoolify.prototype.autoType = function(selector, text, relpos, timeout) {
   }
   return this;
 };
+
+_Xdotoolify.prototype.sleepUntil = function(predicate, ms, immediate) {
+  this.operations.push({
+    type: 'sleepUntil',
+    predicate: predicate || function() {return new Promise(function(resolve) {
```

Comment:
i don't think predicate should have a default value, there's are no use cases where you'd pass in a null predicate

Reply:
Fixed